packages/loader/container-loader/src/memoryBlobStorage.ts

```diff
@@ -10,6 +10,12 @@ import type {
 	IDocumentStorageService,
 } from "@fluidframework/driver-definitions/internal";
 
+/**
+ * An interface used to manage blobs in memory for detached containers.
+ * On attach of the container the blobs are read, and uploaded to the server.
```

Comment:
Nit (since this describes how it works vs what it is)
```suggestion
 * 
 * @remarks
 * On attach of the container the blobs are read, and uploaded to the server.
```